pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -168,6 +171,7 @@ in buildPythonPackage {
         "$out/${python.sitePackages}/tensorflow/python/saved_model"
         "$out/${python.sitePackages}/tensorflow/python/util"
         "$out/${python.sitePackages}/tensorflow/tsl/python/lib/core"
+        "$out/${python.sitePackages}/tensorflow.libs/"
```

Comment:
A note for later: I think it's best that we get rid of the custom fixup logic. There are many other places in nixpkgs where implement a similar logic, which was the motivation for the now-abandoned https://github.com/NixOS/nixpkgs/pull/297590